src/content/developers/docs/scaling/plasma/index.md

```diff
@@ -18,9 +18,31 @@ You should have a good understanding of all the foundational topics and a high-l
 | High throughput, low cost per transaction.                                                                            | Does not support general computation. Only basic token transfers, swaps, and a few other transaction types are supported via predicate logic.                               |
-| Good for transactions between arbitrary users (no overhead per user pair if both are established on the plasma chain). | Need to periodically watch the network (liveness requirement) or delegate this responsibility to someone else to ensure the security of your funds.                         |
-|                                                                                                                       | Relies on one or more operators to store data and serve it upon request.                                                                                                    |
-|                                                                                                                       | Withdrawals are delayed by several days to allow for challenges. For fungible assets this can be mitigated by liquidity providers, but there is an associated capital cost. |
+| Good for transactions between arbitrary users (no overhead per user pair if both are established on the plasma chain) | Need to periodically watch the network (liveness requirement) or delegate this responsibility to someone else to ensure the security of your funds.                         |
+|  Plasma chains can be adapted to specific use-cases that are unrelated to the main chain. Anyone, including businesses, can customize Plasma smart contracts to provide scalable infrastructure that works in different contexts.                                                       | Relies on one or more operators to store data and serve it upon request.                                                                                              |
+|                                                                                                                        | Withdrawals are delayed by several days to allow for challenges. For fungible assets this can be mitigated by liquidity providers, but there is an associated capital cost. |
+|                                                                                                                        | If too many users try to exit their Plasma chain simultaneously (due to malicious activity or network failure), they could flood the root chain and congest the network. | 
+
+## How does Plasma differ from sidechains and sharding?
+
+Plasma, sidechains, and sharding are fairly similar because they all connect to Ethereum Mainnet in some way. However, the level and strength of these connections vary, which affects the security properties of each scaling solution. 
+
+Here's a quick comparison of plasma chains, sidechains, and shard chains: 
+
```

Comment:
Not crazy about the tone here (docs tone generally is quite authoritative). I think we can remove this and let headers guide users without losing any clarity. What do you think?

```suggestion
```

Reply:
Sure, we could remove it. I'm used to writing website content, so some of the language carried over. I may have used that line in other places, too, but I'll keep it out of new content.